recipes/torchcam/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "torchcam" %}
+{% set version = "0.3.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/torchcam-{{ version }}.tar.gz
+  sha256: 400d451cbbafe0dbddbf3a22feab9e23d1317e90cf6eae16bca96e0e2ff4f076
+
+build:
+  number: 0
+  noarch: python
+  script: 
+    ## Show the rendered recipe
+    - echo -e "\n>>> Show recipe '{{ name }}/meta.yaml' <<<\n\n"
+    - echo -e $(cat $RECIPE_DIR/meta.yaml)  # [not win]
+    ## Ensure inclusion of requirements.txt
```

Comment:
Can you elaborate a but why exactly you are adding this part?

Reply:
Just to view what the recipe renders to. It will be removed before merging.

Reply:
This was about the `## Ensure inclusion of requirements.txt` line. I'm not sure what you're doing there. If `requirements.txt` is not there the build will fail and it should be permanently added. If it is there, this wouldn't be needed, if I understand it correctly?